java-samples/corda5-negotiation-cordapp/workflows/build.gradle

```diff
@@ -38,15 +41,44 @@ dependencies {
 
     // 3rd party libraries
     // Required
-    testImplementation "org.slf4j:slf4j-simple:2.0.0"
+    testImplementation "org.slf4j:slf4j-simple:1.7.36"
+    testRuntimeOnly "org.slf4j:slf4j-simple:1.7.36"
```

Comment:
I don't think you need the second testRuntimeOnly as you have testImplementation